### PR TITLE
PXC-3310 Document wsrep_ignore_apply_errors variable default to resto…

### PR DESCRIPTION
**Pull request by patrickbirch**
…re 5.x behavior

Added wsrep_ignore_apply_errors variable and description.

### CODE REVIEW

**Review by kamil-holubicki — APPROVED** [review]
lgtm

File: doc/source/wsrep-system-index.rst [hunk 1]
```diff
@@ -418,6 +418,64 @@ Possible values for this variable are:
...
+The variable has the following options:
+
+.. list-table::
+    :widths: 25 30
+    :header-rows: 1
+    
+    * - Value
+      - Description
+    * - WSREP_IGNORE_ERRORS_NONE
+      - All replication errors are treated as errors and will shutdown the node or MySQL server (default behavior)
```
Comment (altmannmarcelo) [inline doc/source/wsrep-system-index.rst]: node or MySQL server is the same thing here. We should choose one.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]

**Review by kathy-williamson — APPROVED** [review]

File: doc/source/wsrep-system-index.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/wsrep-system-index.rst]: Fixed.

**Review by altmannmarcelo — APPROVED** [review]
LGTM

**Pull request merged**